src/upgradeable/ERC721PresetMinterPauserAutoIdCustomizedUpgradeable.sol

```diff
@@ -0,0 +1,179 @@
+
+import "../../lib/openzeppelin-contracts-upgradeable/contracts/token/ERC721/ERC721Upgradeable.sol";
+import "../../lib/openzeppelin-contracts-upgradeable/contracts/token/ERC721/extensions/ERC721EnumerableUpgradeable.sol";
+import "../../lib/openzeppelin-contracts-upgradeable/contracts/token/ERC721/extensions/ERC721BurnableUpgradeable.sol";
+import "../../lib/openzeppelin-contracts-upgradeable/contracts/token/ERC721/extensions/ERC721PausableUpgradeable.sol";
+import "../../lib/openzeppelin-contracts-upgradeable/contracts/access/AccessControlEnumerableUpgradeable.sol";
+import "../../lib/openzeppelin-contracts-upgradeable/contracts/utils/ContextUpgradeable.sol";
+import "../../lib/openzeppelin-contracts-upgradeable/contracts/utils/CountersUpgradeable.sol";
+import {Initializable} from "../../lib/openzeppelin-contracts-upgradeable/contracts/proxy/utils/Initializable.sol";
+
+/**
+ * @dev {ERC721} token, including:
+ *
+ *  - ability for holders to burn (destroy) their tokens
+ *  - a minter role that allows for token minting (creation)
+ *  - a pauser role that allows to stop all token transfers
+ *  - token ID and URI autogeneration
+ *
+ * This contract uses {AccessControl} to lock permissioned functions using the
+ * different roles - head to its documentation for details.
+ *
+ * The account that deploys the contract will be granted the minter and pauser
+ * roles, as well as the default admin role, which will let it grant both minter
+ * and pauser roles to other accounts.
+ *
+ * _Deprecated in favor of https://wizard.openzeppelin.com/[Contracts Wizard]._
+ */
+contract ERC721PresetMinterPauserAutoIdCustomizedUpgradeable is
+  Initializable,
+  ContextUpgradeable,
+  AccessControlEnumerableUpgradeable,
+  ERC721EnumerableUpgradeable,
+  ERC721BurnableUpgradeable,
+  ERC721PausableUpgradeable
+{
+  error ErrUnauthorizedAccount(address account, bytes32 neededRole);
+
+  using CountersUpgradeable for CountersUpgradeable.Counter;
+
+  bytes32 public constant MINTER_ROLE = keccak256("MINTER_ROLE");
+  bytes32 public constant PAUSER_ROLE = keccak256("PAUSER_ROLE");
+
+  CountersUpgradeable.Counter private _tokenIdTracker;
+
+  string private _baseTokenURI;
+
+  function initialize(string memory name, string memory symbol, string memory baseTokenURI) public virtual initializer {
+    __ERC721PresetMinterPauserAutoId_init(name, symbol, baseTokenURI);
+  }
+  /**
+   * @dev Grants `DEFAULT_ADMIN_ROLE`, `MINTER_ROLE` and `PAUSER_ROLE` to the
+   * account that deploys the contract.
+   *
+   * Token URIs will be autogenerated based on `baseURI` and their token IDs.
+   * See {ERC721-tokenURI}.
+   */
+
+  function __ERC721PresetMinterPauserAutoId_init(string memory name, string memory symbol, string memory baseTokenURI)
+    internal
+    onlyInitializing
+  {
+    __ERC721_init_unchained(name, symbol);
+    __Pausable_init_unchained();
+    __ERC721PresetMinterPauserAutoId_init_unchained(name, symbol, baseTokenURI);
+    _tokenIdTracker.increment();
```

Comment:
Why this line is not in `_init_unchained` but here? If it `_unchained` it means no need to increase the id tracker?